mesonbuild/optinterpreter.py

```diff
@@ -188,8 +189,11 @@ def evaluate_statement(self, node):
             raise OptionException('Only calls to option() are allowed in option files.')
         (posargs, kwargs) = self.reduce_arguments(node.args)
 
-        if 'yield' in kwargs:
-            FeatureNew('option yield', '0.45.0').use()
+        # FIXME: Cannot use FeatureNew while parsing options because we parse
+        # it before reading options in project(). See func_project() in
+        # interpreter.py
+        #if 'yield' in kwargs:
```

Comment:
__[Flake8]__

[E265] block comment should start with '# '


<!-- Comment by [Sider](https://sider.review) -->